organization/general_rules_and_guidelines.rst

```diff
@@ -0,0 +1,32 @@
+General rules and guidelines
+============================
+
+This page contains guidelines relevant for contributing to any area of Godot.
+
+When contributing to any particular area, please refer to its respective guidelines additionally.
+
+Code of conduct
+---------------
+
+When you contribute to Godot, we expect that you respect our `Code of Conduct <https://godotengine.org/code-of-conduct/>`__.
+
+Contributing with AI and large language models
+----------------------------------------------
+
+The use of AI to contribute to Godot is discouraged.
+We acknowledge that AI can be useful, but we are convinced that effort by humans results in better and more relevant
+contributions.
+
+If you use AI to assist with your contribution, you are required to:
+
+- Understand and verify any of its solutions.
+    - If you don't understand the solution, do not submit it.
+- Proof read and edit anything it generates.
+    - Better, use AI to proof read something *you* create.
+- Disclose how and to what degree you used AI (in the issue description, pull request description, or otherwise).
```

Comment:
"or otherwise"?

Reply:
What you may want to do is explain what "contribution" refers to in this case way beforehand (maybe below _"The use of AI to contribute to Godot is discouraged."_), which is actually quite vague up until these examples in parentheses. Assuming that's done:
```suggestion
- Disclose how and how much you used AI for your contribution.
```

Reply:
That's a great way to phrase it!